### DISCUSSION TITLE
Chocolatey Package -- Question

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#1321] [Q&A]
Sorry if this is not the best place to ask this: but the chocolately package installed ungoogled chromium at say `C:\ProgramData\chocolatey\lib\ungoogled-chromium\tools\ungoogled-chromium_87.0.4280.67-1.1_windows` for example.
I had to make a shortcut to this on my desktop.
My concern is that when ungoogled chromium updates, it will make the shortcut break because the version will change ie `C:\ProgramData\chocolatey\lib\ungoogled-chromium\tools\ungoogled-chromium_89.01_windows`

Is this true? Can I do something about this?

**Comment by yasinkavakli** [comment]
That's strange, I've installed the Chocolatey package and chrome.exe is installed under `C:\ProgramData\chocolatey\bin` for me.
So I could reference it with `C:\ProgramData\chocolatey\bin\chrome.exe` and it also works over updates.

**Reply by chiragwad** [reply]
is the chocolatey package actively updated? because i saw that the version on chocolatey is a few version behind what is on github.

**Reply by yasinkavakli** [reply]
The choco packages takes the downloads from here: https://ungoogled-software.github.io/ungoogled-chromium-binaries/ so it's a little bit behind but not that much.

Not sure if it's automated but the update script can be found here: https://gitlab.com/dimqua/choco/-/blob/stable/ungoogled-chromium/update.ps1

**Reply by ghost** [reply]
@blackbeardth , thanks. It installed chrome.exe there as well for me. I guess it shortcuts the program there but keeps the program in the path I said above. This solves my issue however, as I only need to use the link you pointed out. Thanks so much!

**Reply by chiragwad** [reply]
@Coupz i don't mind if the package is one or two version old, just wanted to know if it gets regular updates like once or twice a month

**Reply by chiragwad** [reply]
![Screenshot (14)](https://user-images.githubusercontent.com/70256225/104479931-fa4ca500-55e9-11eb-81a6-fb6858bddf1e.png)

for some reason when i use   C:\ProgramData\chocolatey\bin\chrome.exe   to create shotcut on dektop the icon is very pixelated and not sharp , but when i use C:\ProgramData\chocolatey\lib\ungoogled-chromium\tools\ungoogled-chromium_87.0.4280.67-1.1_windows\chrome.exe   to create icon , it is fine. anyone knows why?

**Reply by ghost** [reply]
@blackbeardth actually I just noticed I have the same issue as well.
